pkg/yurthub/metrics/metrics.go

```diff
@@ -0,0 +1,132 @@
+/*
+Copyright 2020 The OpenYurt Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package metrics
+
+import (
+	"strings"
+
+	"github.com/openyurtio/openyurt/pkg/projectinfo"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var (
+	namespace = "node"
+	subsystem = strings.ReplaceAll(projectinfo.GetHubName(), "-", "_")
+)
+
+var (
+	// Metrics provides access to all hub agent metrics.
+	Metrics = newHubMetrics()
+)
+
+type HubMetrics struct {
+	serversHealthy         *prometheus.GaugeVec
```

Comment:
Can we somehow make a consistent naming convention here?
For example, XXXCounter means Counter. XXX means Gauge. XXXs means GaugeVec.

        inFlightRequest  prometheus.Gauge
	rejectedRequestCounter prometheus.Counter
	closableConns    *prometheus.GaugeVec

Reply:
fixed